docs/deployment-github-pages.md

```diff
@@ -26,24 +26,20 @@ Navigate to your repository settings, click on Pages and enable GitHub pages. Wh
 
 To trigger action, push your code with the workflow to main.
 
-:::{warning} Custom Domains
-GitHub allows you to host your static content on a custom domain, doing so _may_ require you to change the `BASE_URL` environment variable in the action. If you have unstyled content, try changing the `BASE_URL` to a blank string: `BASE_URL=''` (note the **single quotes**!); this serves the build assets from the root of your domain, rather than the default, which is the name or your repository.
-:::
-
-:::{tip} `BASE_URL` environment variable
-:class: dropdown
-The build and site assets are in the `/build` folder, which would point outside of the current repository to a repository called 'build', which probably doesn't exist!
+:::{tip} You will probably need to set a `BASE_URL`
+GitHub Pages likely requires you to set a `BASE_URL` because the URL for GitHub Pages defaults to a sub-folder of `username.github.io` (e.g. `username.github.io/myrepository/`).
```

Comment:
```suggestion
:::{tip} `BASE_URL` Configuration for GitHub Pages

If you are deploying your mystmd website from a specific repository within an organization that has a domain url setup then you likely need to define a `BASE_URL` that includes the repository name.

For example, if your GitHub organization was connected to mystmd.org and you had a repository  called myrepository where your. mystmd website lives, you'd want the base url to be:

`username.github.io/myrepository/`

GitHub Pages likely requires you to set a `BASE_URL` because the URL for GitHub Pages defaults to a sub-folder of `username.github.io` (e.g. `username.github.io/myrepository/`).
```